### ISSUE TITLE
Tokens aren't included in the theme type definition

### ISSUE THREAD

**Issue by rudeayelo** [modulz/stitches#512]
# Bug report

## Describe the bug

As stated in the documentation, a `theme` object contains information about the defined tokens:

```js
const dark = theme({
  colors: {
    primary: 'white',
  },
});

dark.colors.primary.value; // white
dark.colors.primary.token; // primary
dark.colors.primary.scale; // colors
dark.colors.primary.variable; // --colors-primary
```

that is not reflected in the type definition:

![image](https://user-images.githubusercontent.com/140042/112284062-170bf580-8c89-11eb-8728-821746ad0bfb.png)

## To Reproduce

1. Define a theme `const dark = theme({ colors: { primary: "#333" }})`
2. Try and get a token type from it `dark.colors`
3. TypeScript will complain that `colors` doesn't exist in `dark`

## Expected behavior

Token types are defined, probably within the `ThemeRule` type.

---

Thanks for the great work, Stitches is 🔥 !

**Comment by jonathantneal** [comment]
These will be available in #659 —

```tsx
import { createCss } from '@stitches/react'

const { theme } = createCss({
  theme: {
    colors: {
      red: '#ff0000',
    },
  },
})

console.log(theme.colors.red)
```

<img width="420" alt="image" src="https://user-images.githubusercontent.com/188426/126389344-cb23acf5-f450-4943-9798-1da9b3fff531.png">
<img width="420" alt="image" src="https://user-images.githubusercontent.com/188426/126389400-e6b7b3f1-ba8f-46cd-8c56-f4457fdb2aac.png">

**Comment by peduarte** [comment]
Fixed in V1. Please refer to [migration guide](https://stitches.dev/blog/migrating-from-beta-to-v1) and [changelog](https://stitches.dev/blog/stitches-1.0.0)
